pkg/ccl/crosscluster/logical/dead_letter_queue_test.go

```diff
@@ -105,60 +106,122 @@ func TestDLQClient(t *testing.T) {
 
 	sqlDB := sqlutils.MakeSQLRunner(db)
 	sqlDB.Exec(t, `CREATE TABLE foo (a INT)`)
+	sqlDB.Exec(t, `CREATE TABLE bar (a INT)`)
```

Comment:
could you create bar in a user defined schema? i.e. first create a schema and then create bar in it.